### ISSUE TITLE
Is there any way to add an option separator in a dropdown field using survey JSON?

### ISSUE THREAD

**Issue by bappajubcse05** [surveyjs/surveyjs#1108]
### Are you requesting a feature, reporting a bug or ask a question?

Question

### What is the current behavior?
Is there any way to add an option separator in a dropdown field as shown in the attached image using survey JSON?

![capture](https://user-images.githubusercontent.com/34443377/40231521-f898e410-5a8a-11e8-8a99-ed11f2fc09e5.JPG)

### What is the expected behavior?

Is there any way to add an option separator in a dropdown field as shown in the attached image using survey JSON?

![capture](https://user-images.githubusercontent.com/34443377/40231521-f898e410-5a8a-11e8-8a99-ed11f2fc09e5.JPG)

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser:
* browser version:
* surveyjs platform (angular or react or jquery or knockout or vue):
* surveyjs version:

**Comment by tsv2013** [comment]
No, SurveyJS hasn't such an option. Thank you for the suggestion. We'll keep it in mind.

**Comment by tsv2013** [comment]
I think you can use Select2 custom widget. The sample is here - https://surveyjs.io/Examples/Library/?id=custom-widget-select2&platform=jQuery&theme=default
But probably you will need some coding to set proper options.
